src/ApiCompat/Microsoft.DotNet.ApiCompatibility/Abstractions/Mappers/AssemblyMapper.cs

```diff
@@ -36,20 +45,14 @@ public IEnumerable<NamespaceMapper> GetNamespaces()
                 _namespaces = new Dictionary<INamespaceSymbol, NamespaceMapper>(Settings.EqualityComparer);
                 AddOrCreateMappers(Left, ElementSide.Left);
 
-                if (Right.Length == 1)
-                {
-                    AddOrCreateMappers(Right[0], ElementSide.Right);
-                }
-                else
+                for (int i = 0; i < Right.Length; i++)
                 {
-                    for (int i = 0; i < Right.Length; i++)
-                    {
-                        AddOrCreateMappers(Right[i], ElementSide.Right, i);
-                    }
+                    AddOrCreateMappers(Right[i], ElementSide.Right, i);
```

Comment:
Same clean-up as in https://github.com/dotnet/sdk/commit/73045163de3dab3eb1affb7e18f738d45ddd4e04. The if/else isn't required here.